kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/ParseLog.kt

```diff
@@ -0,0 +1,70 @@
+package kotlinx.fuzz.jazzer
+
+import java.nio.file.Path
+import kotlin.io.path.readLines
+import kotlin.time.Duration
+
+internal data class LibfuzzerLogEntryNoTimestamp(
+    val execNr: Int,
+    val cov: Int,
+    val ft: Int,
+    val crashes: Int,
+) {
+    fun addTimestamp(timeSeconds: Long): LibfuzzerLogEntry = LibfuzzerLogEntry(
+        timeSeconds = timeSeconds,
+        execNr = execNr,
+        cov = cov,
+        ft = ft,
+        crashes = crashes,
+    )
+}
+
+internal data class LibfuzzerLogEntry(
+    val timeSeconds: Long,
+    val execNr: Int,
+    val cov: Int,
+    val ft: Int,
+    val crashes: Int,
+)
+
+internal fun jazzerLogToCsv(file: Path, duration: Duration): String {
+    var crashes = 0
+
+    val statsNoTimeStamps = file.readLines().mapNotNull { line ->
+        val tokens = line.split("\\s+".toRegex())  // Split line into tokens
+        when {
+            tokens.size < 2 -> null
+
+            tokens[0].startsWith("artifact_prefix=") -> {
+                crashes++
+                null
+            }
+
+            !tokens[0].startsWith("#") -> null
+            @Suppress("MAGIC_NUMBER")
+            tokens.size < 14 -> null
```

Comment:
I would at least add some comments in case of someone will try to debug it

Reply:
Added log line examples and link to libffuzzer's doc (good luck future me)